libexec/probes/gfactory-querying-local-probe

```diff
@@ -29,8 +29,8 @@ class GfactoryQueryingLocalProbe(rsvprobe.RSVProbe):
         self.supported_metrics = [metric]
         self.details = "---\n"
         self.gfactory = "glidein-itb.grid.iu.edu"
-        self.constraint = """(GLIDEIN_GridType=?="condor" || GLIDEIN_GridType=?="gt5" || GLIDEIN_GridType=?="gt2") && stringListMember("CMS",GLIDEIN_Supported_VOs)"""
-        self.dummyMetric = "org.osg.general.osg-version"
+        self.constraint = """(GLIDEIN_GridType=?="condor" || GLIDEIN_GridType=?="gt5" || GLIDEIN_GridType=?="gt2" || GLIDEIN_GridType=?="cream" || GLIDEIN_GridType=?="nordugrid") && stringListMember("CMS",GLIDEIN_Supported_VOs)"""
```

Comment:
In `etc/metrics/org.osg.local-gfactory-querying-local.conf`, you changed `stringListMember("CMS",GLIDEIN_Supported_VOs)` to `stringListMember("IceCube",GLIDEIN_Supported_VOs)`. But in `libexec/probes/gfactory-querying-local-probe` I don't see you making the same change -- the constraint still uses "CMS" instead of "IceCube".